index.html

```diff
@@ -24,71 +24,449 @@
+            <li class="js-css-html-button-li">
+              <button type="button" class="html-css-js-button">css</button>
+            </li>
+            <li class="js-css-html-button-li">
+              <button type="button" class="html-css-js-button">
+                javascript
+              </button>
+            </li>
+          </ul>
+
+          <button type="button" class="footer-button">
+            <i class="see-project-text">See Project</i>
+          </button>
+        </section>
+      </div>
+    </div>
+    <div class="grid-item">
+      <div class="grid-item-last-two-pages">
+        <div class="about-myself-div">
+          <h1 class="text-about-myself">About Myself</h1>
+        </div>
+
+        <section>
+          <p class="landing-page-main-body-text for-about-myself-only">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website. Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <h4 class="follow for-about-myself-only">LET'S CONNECT</h4>
+
+          <ul class="social-media-icon-container for-myself-social-media-icon">
+            <li class="social-media-icon">
+              <a
+                href="https://twitter.com/haywayalive"
+                target="_blank"
+                rel="noopener noreferrer"
+                ><i class="fa social-media-icon-color fa-twitter"></i
+              ></a>
+            </li>
+
+            <li class="social-media-icon">
+              <a
+                href="https://www.linkedin.com/in/abubakar-ummar-4b6643245"
+                target="_blank"
+                rel="noopener noreferrer"
+                ><i class="fa social-media-icon-color fa-linkedin"></i
+              ></a>
+            </li>
+
+            <li class="social-media-icon">
+              <a
+                href="https://medium.com/@haywayaheadshotstrategy.fx"
+                target="_blank"
+                rel="noopener noreferrer"
+                ><i class="fa social-media-icon-color fa-medium"></i
+              ></a>
+            </li>
+
+            <li class="social-media-icon">
+              <a
+                href="https://github.com/Haywayaheadshot"
+                target="_blank"
+                rel="noopener noreferrer"
+                ><i class="fa social-media-icon-color fa-github"></i
+              ></a>
+            </li>
+
+            <li class="social-media-icon">
+              <a
+                href="https://angel.co/u/abubakar-ummar"
+                target="_blank"
+                rel="noopener noreferrer"
+                ><i class="fa social-media-icon-color fa-angellist"></i
+              ></a>
+            </li>
+          </ul>
+
+          <button type="button" class="footer-button">
+            <i class="see-project-text">Get my resume</i>
+          </button>
```

Comment:
- [ ] Kindly add the appropriate button  styling for  all buttons in this section, please see the design, and follow the button styling for Active, Hover, and Pressed 

![Buttons](https://user-images.githubusercontent.com/69638013/185978515-ed1ae323-370b-4470-8a4f-e58681583aa0.png)